### PR TITLE
Clear NuGet security warnings

### PR DESCRIPTION
**Pull request by davidmrdavid**
This is the DTFx-MSSQL version of this PR: https://github.com/Azure/azure-functions-durable-extension/pull/2842

As part of the 1ES migration, we have new supply chain requirements around our repo's nuget.config files.

In particular, we're getting errors due to use MyGet. This PR removes our MyGet usage. It also addresses warnings regarding missing `<clear/>` statements.

### CODE REVIEW

File: nuget.config [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 <configuration>
   <packageSources>
+    <clear/>
     <add key="nuget.org" value="https://www.nuget.org/api/v2/" />
-    <add key="azure-appservice-staging" value="https://www.myget.org/F/azure-appservice-staging/api/v3/index.json" />
     <add key="azure-appservice" value="https://www.myget.org/F/azure-appservice/api/v3/index.json" />
```
Comment (jviau) [inline nuget.config:7]: this will need to be removed or replaced with an ADO feed as well.

**Review by jviau — CHANGES_REQUESTED** [review]

File: nuget.config (hunk 1 above)
Comment (davidmrdavid) [reply nuget.config:7]: ah, I missed there were two of these. Fixing promptly

Comment (davidmrdavid) [reply nuget.config]: @jviau - fixed in next commit

Comment (davidmrdavid) [reply nuget.config]: actually, I guess that if we only have `nuget.org`, we don't really need a `nuget.config`....but I think we'll eventually want to add the ADO feed so I'll keep it around

Comment (jviau) [reply nuget.config]: You are expected to always have a `nuget.config`, because the `<clear />` element is important.

**Review by jviau — APPROVED** [review]

**Pull request merged**